operate/client/src/modules/queries/decisionInstances/useDrdData.ts

```diff
@@ -0,0 +1,49 @@
+/*
+ * Copyright Camunda Services GmbH and/or licensed to Camunda Services GmbH under
+ * one or more contributor license agreements. See the NOTICE file distributed
+ * with this work for additional information regarding copyright ownership.
+ * Licensed under the Camunda License 1.0. You may not use this file
+ * except in compliance with the Camunda License 1.0.
+ */
+
+import {useQuery} from '@tanstack/react-query';
+import {queryKeys} from '../queryKeys';
+import {searchDecisionInstances} from 'modules/api/v2/decisionInstances/searchDecisionInstances';
+import type {DecisionInstance} from '@camunda/camunda-api-zod-schemas/8.8';
+
+interface DrdData {
+  [decisionDefinitionId: DecisionInstance['decisionDefinitionId']]: {
+    decisionDefinitionId: DecisionInstance['decisionDefinitionId'];
+    decisionEvaluationInstanceKey: DecisionInstance['decisionEvaluationInstanceKey'];
+    state: DecisionInstance['state'];
+  };
+}
+
+const useDrdData = (decisionEvaluationKey?: string) => {
+  return useQuery({
+    enabled: !!decisionEvaluationKey,
+    queryKey: queryKeys.decisionInstances.drdData(decisionEvaluationKey ?? ''),
+    queryFn: async () => {
+      const {response, error} = await searchDecisionInstances({
```

Comment:
The search endpoints has paging while the old one did not. What is the expected amount of matching items? Do we need to handle paging?

If we need to handle paging, how should it be handled? One idea is to make one request, and based on the page information, trigger requests for the remaining items to combine them all.

Reply:
@christoph-fricke we must handle pagination. I think we can simply handle in a single `queryFn` and just cache it

Do you think it's better to handle it on this PR or on a follow-up?